showcase/src/menu/menu.c

```diff
@@ -108,11 +108,11 @@ void gsMenuDestroy(void) {
 
 void menuDrawBG() {
 	UWORD uwX, uwY;
-	UBYTE ubOdd, ubColor;
+	UBYTE ubOdd = 0, ubColor;
```

Comment:
This line could be actually tricky. I'm not a big fan of declarations multiple variables in one line. Highlighted line can be error prone when assignment would be after ubColor. Someone can think of assigning same value to all declarations, which is not true.

Would totally go for single declaration per line.

Reply:
Fixed in 52602be